src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/InputSeparatorWrapForTestComma2.java

```diff
@@ -0,0 +1,28 @@
+/*
+SeparatorWrap
+tokens = COMMA
+option = nl
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.separatorwrap;
+
+public class InputSeparatorWrapForTestComma2 {
+
+    String s;
+
+    // violation below '',' should be on a new line.'
```

Comment:
Please add some additional comments above these to mention that there is trailing whitespace after `,` token

Reply:
Done